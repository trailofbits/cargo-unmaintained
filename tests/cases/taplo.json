[
  {
    "name": "adler",
    "outdated_deps": [],
    "repo_status": "Archived",
    "version": "1.0.2"
  },
  {
    "name": "dirs",
    "outdated_deps": [],
    "repo_status": "Archived",
    "version": "1.0.5"
  },
  {
    "name": "hermit-abi",
    "outdated_deps": [],
    "repo_status": "Unassociated",
    "version": "0.1.19"
  },
  {
    "name": "proc-macro-error",
    "outdated_deps": [
      {
        "name": "syn",
        "req": "^1",
        "version_latest": null,
        "version_used": "1.0.103"
      }
    ],
    "repo_status": {
      "Age": null
    },
    "version": "1.0.4"
  },
  {
    "name": "rust-argon2",
    "outdated_deps": [
      {
        "name": "base64",
        "req": "^0.13",
        "version_latest": null,
        "version_used": "0.13.1"
      },
      {
        "name": "blake2b_simd",
        "req": "^0.5",
        "version_latest": null,
        "version_used": "0.5.11"
      },
      {
        "name": "constant_time_eq",
        "req": "^0.1.4",
        "version_latest": null,
        "version_used": "0.1.5"
      }
    ],
    "repo_status": {
      "Age": null
    },
    "version": "0.8.3"
  },
  {
    "name": "serde_cbor",
    "outdated_deps": [],
    "repo_status": "Archived",
    "version": "0.11.2"
  },
  {
    "name": "structopt-derive",
    "outdated_deps": [
      {
        "name": "heck",
        "req": "^0.3.0",
        "version_latest": null,
        "version_used": "0.3.3"
      },
      {
        "name": "syn",
        "req": "^1",
        "version_latest": null,
        "version_used": "1.0.103"
      }
    ],
    "repo_status": {
      "Age": null
    },
    "version": "0.4.18"
  },
  {
    "name": "structopt",
    "outdated_deps": [
      {
        "name": "clap",
        "req": "^2.33",
        "version_latest": null,
        "version_used": "2.34.0"
      }
    ],
    "repo_status": {
      "Age": null
    },
    "version": "0.3.26"
  },
  {
    "name": "tinytemplate",
    "outdated_deps": [
      {
        "name": "criterion",
        "req": "^0.3",
        "version_latest": null,
        "version_used": "0.3.6"
      }
    ],
    "repo_status": {
      "Age": null
    },
    "version": "1.2.1"
  },
  {
    "name": "tokio-rustls",
    "outdated_deps": [],
    "repo_status": "Unassociated",
    "version": "0.23.4"
  }
]
