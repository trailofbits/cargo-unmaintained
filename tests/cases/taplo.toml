url = "https://github.com/tamasfe/taplo"
rev = "0.8.1"
use_real_github = true
