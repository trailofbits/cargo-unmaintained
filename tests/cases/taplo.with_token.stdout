lsp-async-stub (no repository)
    lsp-types (requirement: ^0.93.0, version used: 0.93.2, latest: [..])
taplo-common (no repository)
    ahash (requirement: ^0.7.6, version used: 0.7.6, latest: [..])
    indexmap (requirement: ^1.6.0, version used: 1.9.1, latest: [..])
    itertools (requirement: ^0.10.3, version used: 0.10.5, latest: [..])
    json_value_merge (requirement: ^1.1.2, version used: 1.1.2, latest: [..])
    lru (requirement: ^0.7.2, version used: 0.7.8, latest: [..])
arc-swap (https://github.com/vorner/arc-swap updated [..] days ago)
    criterion (requirement: ~0.3, version used: 0.3.6, latest: [..])
    itertools (requirement: ^0.10, version used: 0.10.5, latest: [..])
prettytable-rs (https://github.com/phsym/prettytable-rs updated [..] days ago)
    encode_unicode (requirement: ^0.3, version used: 0.3.6, latest: [..])
    term (requirement: ^0.5, version used: 0.5.2, latest: [..])
hex (https://github.com/KokaKiwi/rust-hex updated [..] days ago)
    criterion (requirement: ^0.3, version used: 0.3.6, latest: [..])
tinytemplate (https://github.com/bheisler/TinyTemplate updated [..] days ago)
    criterion (requirement: ^0.3, version used: 0.3.6, latest: [..])
debugid (https://github.com/getsentry/rust-debugid updated [..] days ago)
    uuid (requirement: ^0.8.1, version used: 0.8.2, latest: [..])
proc-macro-error (https://gitlab.com/CreepySkeleton/proc-macro-error updated [..] days ago)
    syn (requirement: ^1, version used: 1.0.103, latest: [..])
atty (https://github.com/softprops/atty updated [..] days ago)
    hermit-abi (requirement: ^0.1.6, version used: 0.1.19, latest: [..])
adler (https://github.com/jonas-schievink/adler.git updated [..] days ago)
    criterion (requirement: ^0.3.2, version used: 0.3.6, latest: [..])
matchers (https://github.com/hawkw/matchers updated [..] days ago)
    regex-automata (requirement: ^0.1, version used: 0.1.10, latest: [..])
regex-automata (https://github.com/BurntSushi/regex-automata archived)
serde_cbor (https://github.com/pyfisch/cbor archived)
toml (https://github.com/alexcrichton/toml-rs archived)
